bundle/edu.gemini.ags/src/main/java/edu/gemini/ags/gems/GemsGuideStarSearchOptions.java

```diff
@@ -64,9 +65,9 @@ public String toString() {
 
 
     public enum NirBandChoice {
-        J(GemsUtils4Java.toNewBand(edu.gemini.shared.skyobject.Magnitude.Band.J)),
-        H(GemsUtils4Java.toNewBand(edu.gemini.shared.skyobject.Magnitude.Band.H)),
-        K(GemsUtils4Java.toNewBand(edu.gemini.shared.skyobject.Magnitude.Band.K)),
+        J(ModelConverters.toNewBand(edu.gemini.shared.skyobject.Magnitude.Band.J)),
+        H(ModelConverters.toNewBand(edu.gemini.shared.skyobject.Magnitude.Band.H)),
+        K(ModelConverters.toNewBand(edu.gemini.shared.skyobject.Magnitude.Band.K)),
```

Comment:
I guess we could just directly reference the new bands?  Or maybe you're just trying to avoid the awkward `$MODULE` business?